media-fonts/cascadia-code/cascadia-code-2102.03.ebuild

```diff
@@ -0,0 +1,18 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+FONT_SUFFIX="ttf"
```

Comment:
Please move it after BDEPEND.

Reply:
I normally like to put eclass variables before the inherit in case they use them outside of a specific phase but at inherit-time.

Reply:
Yeah, but AFAIK, with font class, it's supposted to be but after all stuff.